tests/test_compute.py

```diff
@@ -89,7 +89,8 @@ def test_compute(client):
         'serviceAgreementId': agreement_id,
         'consumerAddress': cons_acc.address,
         'algorithmDid': alg_ddo.did,
-        'algorithmMeta': {}
+        'algorithmMeta': {},
+        'output': build_stage_output_dict(dict())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'provider_account' in function call](https://app.codacy.com/manual/ocean-protocol/brizo/pullRequest?prid=4927972)